frontend/src/metabase/core/components/Markdown/Markdown.styled.tsx

```diff
@@ -1,10 +1,22 @@
 import type { FC, ReactElement } from "react";
 import ReactMarkdown from "react-markdown";
 import styled from "@emotion/styled";
+import { css } from "@emotion/react";
 import { color } from "metabase/lib/colors";
 import type { MarkdownProps } from "./Markdown";
 
 export const MarkdownRoot = styled(getComponent(ReactMarkdown))<MarkdownProps>`
+  ${props =>
+    props.lineClamp &&
+    css`
+      display: -webkit-box;
+      -webkit-line-clamp: ${props.lineClamp};
```

Comment:
does plain `line-clamp` not work here? will this work in firefox?

Reply:
It has better support and the `-webkit` prefix is oddly part of the spec to account for legacy compatibility (https://drafts.csswg.org/css-overflow-4/#webkit-line-clamp).